apps/docs/src/app/platform/component-docs/platform-wizard-generator/platform-wizard-generator-header/platform-wizard-generator-header.component.html

```diff
@@ -19,6 +19,26 @@ <h3>Platform specific enhancements:</h3>
         For more information, please see
         <a routerLink="/platform/form-generator" fragment="default-configuration">Form generator documentation</a>.
     </p>
+
+    <h3>Wizard Generator Elements:</h3>
+    <ul>
+        <li><b>input</b>: specifies an input field where the user can enter data.</li>
```

Comment:
I'd say these elements are belonging to the form generator documentation, since Wizard generator is based on it. I would move this documentation to form generator page, and simply provide a link to it in wizard generator documentation